cmake/Codegen.cmake

```diff
@@ -165,6 +166,7 @@ if(INTERN_BUILD_ATEN_OPS)
         ${CMAKE_BINARY_DIR}/aten/src/ATen/generated_${gen_type}.cmake
         ${CMAKE_BINARY_DIR}/aten/src/ATen/ops_generated_${gen_type}.cmake
         ${CMAKE_BINARY_DIR}/aten/src/ATen/core_generated_${gen_type}.cmake
+        ${CMAKE_BINARY_DIR}/aten/src/ATen/cpu_vec_generated_${gen_type}.cmake
```

Comment:
This only registers the `.cmake` file as an output, but you also need to add the file contents which is in the variable `${cpu_vec_generated_${gen_type}}`.

Reply:
WHOOPS